flowr/examples/game-of-life/DESCRIPTION.md

```
@@ -0,0 +1,32 @@
+
+Description
+===
+A flow that implements Conway's Game of Life. It takes a grid size, number of
+iterations, and a seed pattern name as arguments. Each generation is rendered
+to an image, showing the evolution of the cellular automaton.
+
+Supported seed patterns: `glider`, `blinker`, `block`, `rpentomino`.
+
+Root Diagram
+===
+<a href="root.dot.svg" target="_blank"><img src="root.dot.svg"></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)